webclient/components/pt-info/single/1time-Mrow-1Field/service-statement/change-layer/c-ct.vue

```diff
@@ -167,6 +173,70 @@ export default {
       }
       return pArOfObjectsFromClientSideMasterDB
     },
+    mfApplyTotalMinutesInPsychotherapyRuleOnServiceStatementCategory(
```

Comment:
When I read the function I have to spend a lot of mental energy to understand what it is doing.

Add doc.

So that I do not have to spend the mental energy to see what pieces of code inside this function are doing.